### PR TITLE
Dashboard Routing

### PR DESCRIPTION
**Pull request by gtoledorodriguez**
Added routing for dashboard in order to preview components

### CODE REVIEW

File: Frontend/src/components/Layout/DashboardPage/Layout.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+const Layout = () => {
+  return (
+    <div >
+      <Header />
+      <main >
+        <Outlet />
+      </main>
+      {/* <Footer /> */}
+    </div>
```
Comment (FevenSeyfu) [inline Frontend/src/components/Layout/DashboardPage/Layout.jsx]: Please update the layout for dashboard to incorporate the sidebar's on the left and right side as well
 <div className="flex flex-col h-screen">
      <div className="flex flex-1 overflow-hidden">
        <aside className="w-20 border overflow-y-auto">
          <LeftSideBar />
        </aside>
        <div className="flex flex-col flex-1">
          <Header className="w-auto" />
          <main className="flex-1 overflow-y-auto">
            <Outlet />
          </main>
        </div>
        <aside className="w-20 border overflow-y-auto">
          <RightSideBar />
        </aside>
      </div>
    </div>

File: Frontend/src/components/Layout/DashboardPage/Layout.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Outlet } from "react-router-dom";
+import Header from "../../Dashboard/Header/Header";
+// import Footer from "../../LandingPage/Footer/Footer";
```
Comment (FevenSeyfu) [inline Frontend/src/components/Layout/DashboardPage/Layout.jsx]: import LeftSideBar from "../../Dashboard/Sidebar/LeftSideBar";
import RightSideBarkfrom "../../Dashboard/Sidebar/RightSideBar";

File: Frontend/src/Pages/Dashboard/Dashboard.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import Header from "../../components/Dashboard/Header/Header.jsx";
+
+const Dashboard = () => {
+    return (
+      <div className="">
+          <Header />
```
Comment (FevenSeyfu) [inline Frontend/src/Pages/Dashboard/Dashboard.jsx]: No need to include the header on the page since it is already added by the Layout.

**Review by FevenSeyfu — CHANGES_REQUESTED** [review]
Hi @gtoledorodriguez ,
Thank you for adding a layout for the portal, after reviewing the changes please make note of the  below suggestion,

- Please resolve the current conflict with main to reflect the current changes in route
- You have done great job on adding the  Layout for the portal However, please note we have 2 sidebars on the left and right sidebar

File: Frontend/src/components/Layout/DashboardPage/Layout.jsx (hunk 1 above)
Comment (gtoledorodriguez) [reply Frontend/src/components/Layout/DashboardPage/Layout.jsx]: Added; Left and Right Sidebar Code is temporary will finish tomorrow (Monday) during day

File: Frontend/src/components/Layout/DashboardPage/Layout.jsx (hunk 2 above)
Comment (gtoledorodriguez) [reply Frontend/src/components/Layout/DashboardPage/Layout.jsx]: Added

File: Frontend/src/Pages/Dashboard/Dashboard.jsx (hunk 3 above)
Comment (gtoledorodriguez) [reply Frontend/src/Pages/Dashboard/Dashboard.jsx]: Removed

**Review by FevenSeyfu — APPROVED** [review]
Great work on setting up the routing and layout for the dashboard.

**Pull request merged**